beginners day unconferece

dariagrudzien opened the pull request:
Add a detailed plan of the Beginners' Day Unconference, a part of the Sprints weekend.

The page looks like this: https://ep2025-unconference.ep-preview.click/beginners-day-unconference/

But I'm not able to add it to the menu 🤔

dariagrudzien committed:
beginners day unconferece

europython-society commented:
# Preview available
|Key|Value|
|---|---|
|url|https://ep2025-unconference.ep-preview.click|
|last update|2025-06-16T18:12:54.047Z|

dariagrudzien commented:
@clytaemnestra I've been trying to add Jodie as a reviewer, but I wasn't able to. Is there some way?

nikoshell commented:
@t-redactyl ping

t-redactyl commented on src/content/pages/beginners-day-unconference.mdx:
```diff
@@ -0,0 +1,60 @@
+
+# EuroPython Beginners' Day Unconference
+
+As part of this year's Beginners' Day, on **Saturday 19th July** we'll be hosting the unconference. It's a full day of panel discussions, Q&A sessions, interactive sessions, and short talks focused on getting started and growing as a junior in a tech role. The event will take place at the [Prague University of Economics and Business](https://ep2025.europython.eu/venue/#sprints-venue--saturday--su ...
```
I'd suggest changing

> we'll be hosting the unconference

to 

> we'll be hosting the Unconference

to keep it consistent with the other usages.

t-redactyl reviewed (commented):
I have one very minor suggestion, which you're free to change or ignore (it's just a stylistic change).

I had a look and I can't for the life of me work out why this page isn't being added to the menu: it's in the correct place in links.json. The only thing that maybe looked a bit weird is that it's under "Social & Networking Events" in `overview.astro`, but I have no idea if that makes a difference.

clytaemnestra commented:
@dariagrudzien, @t-redactyl we usually add everything to the menu and to the footer as well (for SEO purposes). 

Looking at this PR, there's an entry for the footer:
<img width="502" alt="image" src="https://github.com/user-attachments/assets/08767ecf-4278-4aec-a5b4-0ac3b221dba7" />

but it's missing for the menu. This part should be updated and then it will appear up there as well 🙂 
<img width="487" alt="image" src="https://github.com/user-attachments/assets/119c6f30-b4a2-411e-a716-84e728a27035" />

clytaemnestra reviewed (commented):
Looks great! Thanks for the PR and for preparing everything! 

I have a few small ideas from my side. Feel free to ignore, if it doesn't align with your vision. These are just my random ideas.

1. What do you think about labeling the event as "In cooperation with Junior Guru"? It's on my list to contact Honza for official media partnership, but since he already advertises it, it might be good for him that JG is a partner of a big, international conference; and it might be easier to sell it for us because local people know JG. We can add JG to official media partners later, now we can put the JG logo at the beginning and advertise it as "JG & EP present: The Unconference Day" (something along those lines). 
2. If I list down the page, it's full of text. I think it's good that every event is explained, but if people have never been at EP or any other unconference event, they might not feel the vibe from the text only and not know what to expect. We have lots of photos from Django Girls or Humble Data from previous years. It's different in a way that it's workshops, and it looks like that, but maybe we can reuse some photos ?

egeakman commented:
@clytaemnestra @dariagrudzien @t-redactyl any updates on this? Can we merge it?

clytaemnestra commented:
I added the label do not merge. It's not ready yet, waiting for the tickets page to be finalized.

t-redactyl commented:
I believe we're just waiting on @dariagrudzien to:
* Add the photos
* Add the reference to Junior Guru.

As per a discussion with Martin, we won't be adding the ticket link to this page any more - it will only be sent out through email once the attendee is accepted to a track.

dariagrudzien committed:
unconference updates

dariagrudzien committed:
junior guru logo fix

dariagrudzien committed:
update the unconference button

dariagrudzien committed:
move unconference in events

dariagrudzien commented:
@t-redactyl @clytaemnestra Updated PR as we discussed, addressing your reviews. Please in particular check:
- junior.guru logo and reference
- positioning of the unconference in the footer and events (I put them apart from Beginners' Day because otherwise it looked like a copy-paste mistake, but open to changing)
- wording around signup and the button

If you'd like to see some changes, please feel free to push directly into the PR and merge at will in the interest of time. Thank you for your patience!

clytaemnestra commented on src/content/pages/beginners-day-unconference.mdx:
```diff
@@ -0,0 +1,86 @@
+
+# EuroPython & junior.guru Beginners' Day Unconference
+
+As part of this year's Beginners' Day, on **Saturday 19th July** we'll be hosting the Unconference in cooperation with [junior.guru](https://junior.guru/). It's a full day of panel discussions, Q&A sessions, interactive sessions, and short talks focused on getting started and growing as a junior in a tech role. The event will take place at the [Prague University of Economics and Business](https:/ ...
```
this hasn't been confirmed yet:  

```suggestion
As part of this year's Beginners' Day, on **Saturday 19th July** we'll be hosting the Unconference in cooperation with [junior.guru](https://junior.guru/). It's a full day of panel discussions, Q&A sessions, interactive sessions, and short talks focused on getting started and growing as a junior in a tech role. 
```

clytaemnestra committed:
Update src/content/pages/beginners-day-unconference.mdx

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

nikoshell committed:
Update images

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

clytaemnestra committed:
Merge branch 'ep2025' into ep2025-unconference
